sdk/core/core/src/az_json_builder.c

```diff
@@ -200,9 +200,7 @@ az_json_builder_write(az_json_builder * const self, az_json_token const value) {
       self->need_comma = true;
       return az_json_builder_write_span(self, value.data.span);
     }
-    default: {
-      return AZ_ERROR_ARG;
-    }
+    default: { return AZ_ERROR_ARG; }
```

Comment:
Is this a contract violation that can return error, or break into the debugger, or be ignored?
If so, this code doesn't accomplish all of this.

Reply:
@sergey-shandar  would have an answer here

Reply:
https://github.com/Azure/azure-sdk-for-c/issues/290